docs/install-w-machine.md

```diff
@@ -177,7 +177,7 @@ Here, you connect to the cluster and review information about the Swarm manager
         .
         .
 
-    In this case, the Swarm ran 'hello-world' on the 'swarm1'.
+    In this case, the Swarm ran 'hello-world' on the 'agent1'.
```

Comment:
While you're editing this line, could you also update the hello-world and agent1 to markdown syntax using backticks?
Like `hello-world` and `agent1`. Thanks!